src/github/nag.rs

```diff
@@ -143,6 +158,26 @@ fn update_proposal_review_status(proposal_id: i32) -> DashResult<()> {
     Ok(())
 }
 
+fn remove_issue_label(issue: &Issue, label: &str) {
```

Comment:
I think these might be a bit clearer if they were methods on Issue, rather than free fns